src/app/pages/jails/jail-add/jail-add.component.ts

```diff
@@ -118,538 +127,644 @@ export class JailAddComponent implements OnInit {
- target="_blank">getfsstat(2)</a>, and <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=fhstatfs&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_blank">fhstatfs(2)</a> and other similar compatibility syscalls.\
- <br>When set to <i>0</i>, all mount points are available without any\
- restrictions.  When set to <i>1</i>, only mount points below the jail\
- chroot directory are visible. When set to <i>2</i>, the syscalls above\
- can operate only on a mountpoint where the jail chroot directory is\
- located.'),
-      options: [{
-        label: 'O',
-        value: '0',
-      }, {
-        label: '1',
-        value: '1',
-      }, {
-        label: '2',
-        value: '2',
-      }]
-    }, {
+      tooltip: T('Determine which information processes in a jail are \
+                 able to obtain about mount points. The behavior of \
+                 multiple syscalls is affected. <a \
+href="https://www.freebsd.org/cgi/man.cgi?query=statfs&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
+target="_blank">statfs(2)</a>, <a \
+href="https://www.freebsd.org/cgi/man.cgi?query=statfs&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
+target="_blank">fstatfs(2)</a>, <a \
+href="https://www.freebsd.org/cgi/man.cgi?query=getfsstat&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
+target="_blank">getfsstat(2)</a>, <a \
+href="https://www.freebsd.org/cgi/man.cgi?query=fhstatfs&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
+target="_blank">fhstatfs(2)</a>, and other similar compatibility syscalls. \
+                 <br>Set to <i>0</i>: All mount points are available \
+                 without restriction. <br>\
+                 Set to <i>1</i>: Only mount points below the jail \
+                 chroot directory are available. <br>\
+                 Set to <i>2</i> (default): Only mounts point where the \
+                 jail chroot directory is located are available.'),
+        options: [{
+            label: '0',
+            value: '0',
+        }, {
+            label: '1',
+            value: '1',
+        }, {
+            label: '2 (default)',
+            value: '2',
+        }]
+    },
+    {
       type: 'input',
       name: 'children_max',
-      placeholder: T('children_max'),
-      tooltip: T('Enter the number of child jails allowed to be created by\
- this jail (or by other jails under this jail). This limit is <i>0</i> by\
- default, indicating the jail is not allowed to create child jails.'),
-    }, {
-      type: 'input',
-      name: 'login_flags',
-      placeholder: T('login_flags'),
-      tooltip: T('List any flags to be passed to <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=login&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_">login(1)</a> when logging in to jails with the <b>console</b>\
- function.'),
+      placeholder: T('children.max'),
+      tooltip: T('Number of child jails allowed to be created by the \
+                 jail or other jails under this jail. The limit is \
```

Comment:
Sentence rework: "The limit is <i>0</i> by default indicating the jail is not allowed to create child jails." -> A limit of <i>0</i> restricts the jail from creating child jails.

Reply:
fixed